tests/cloud_health_api_test.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import unittest
+import sys
+
+#
+# Third party libraries
+#
+
+from mock import MagicMock, patch
+
+#
+# Internal libraries
+#
+
+from krux_cloud_health.cloud_health_api import Application, main
+from krux_cloud_health.cloud_health import Interval, NAME
+from krux.stats import DummyStatsClient
+
+
+class CloudHealthAPITest(unittest.TestCase):
```

Comment:
Please test following in this test case:

1. The return value from `cloud_health.cost_history()` is logged as debug (L68)
1. The category and price for the set time is added to stats. (L84)